mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -186,7 +186,12 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
      */
     fun Completable.execute(
         stateReducer: S.(Async<Unit>) -> S
-    ) = toSingle { Unit }.execute(stateReducer)
+    ) = toObservable<Unit>().execute(stateReducer)
```

Comment:
Also changed `toSingle { Unit }` to `toObservable`. Single version already calls the same one.